### PR TITLE
refactor: cobra.core.model

### PR DESCRIPTION
**Pull request by synchon**
* [ ] fix #(issue number)
* [x] description of feature/fix
* [x] tests added/passed
* [x] add an entry to the [next release](../release-notes/next-release.md)

This PR adds a couple of minor changes to the ones made in #1223 . It was a bit of a hassle to get it there, hence a separate PR.

### CODE REVIEW

File: release-notes/next-release.md [hunk 1]
```diff
@@ -22,3 +22,5 @@ of SBO:0000633 (see https://sourceforge.net/p/sbo/term-request/113/)
 * Removed `model.add_reaction()` and replaced remaining usages of it with `model.add_reactions()`
 * Removed the following tests: test_add_remove_reaction_benchmark, test_add_reaction, test_add_reaction_context, test_add_reaction_from_other_model, test_add_cobra_reaction
 * Removed `model.__add__` and `model.__iadd__` - use `model.merge` to replace them.
+* Remove `Model().description()`.
```
Comment (cdiener) [inline release-notes/next-release.md:25]: Was this done in this PR? I don't see a code change to that effect.

Comment (cdiener) [reply release-notes/next-release.md:25]: Oh I see, that's from the other PR right?

Comment (synchon) [reply release-notes/next-release.md:25]: Right, this one was undocumented.

**Review by cdiener — APPROVED** [review]
:+1:

**Pull request merged**